src/Pyz/Zed/Stock/StockDependencyProvider.php

```diff
@@ -7,9 +7,9 @@
 
 namespace Pyz\Zed\Stock;
 
+use \Spryker\Zed\ProductBundle\Communication\Plugin\Stock\ProductBundleAvailabilityHandlerPlugin;
```

Comment:
Thx, not sure how that came in here
There is still an open issue for it @ https://github.com/spryker/code-sniffer/issues/59
In case you are feeling the burn :P